app/Resources/views/subscription/tabs/tab-metadata.html.twig

```diff
@@ -1,24 +1,30 @@
 <fieldset>
     <div class="row">
         <div class="col-xs-11">
+            {{ form_row(form.importUrl) }}
             {{
                 form_row(
-                    form.metadataUrl,
+                    form.import,
                     {
-                        attr: {
-                            'class': 'url-validated',
-                            'data-parsley-remote': path(
-                                'validate',
-                                {id: subscription.id}
-                            ),
-                            'data-parsley-remote-options': '{ "type": "POST" }',
-                            'data-parsley-errors-messages-disabled': 'true'
+                        'attr': {
+                            'data-requestedState': 'import'
                         }
                     }
                 )
             }}
         </div>
-
+        <div class="col-xs-1">
+            {{
+                form_widget(
+                    form.import,
```

Comment:
Werkt dit? Doet een form_row ook niet een form_widget renderen? En is het niet zo dat een widget maar 1x per form gerendered kan worden?